app/views/layouts/application.html.erb

```diff
@@ -24,16 +24,25 @@
               <a class="nav-link active" aria-current="page" href="#">Home</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="#">Features</a>
+              <a class="nav-link" href="#">My Recipes</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="#">Pricing</a>
+              <a class="nav-link" href="#">Ingredients</a>
```

Comment:
These links should be hidden if the user is not logged in (hint: `if user_signed_in?`)